lib/google-blog-feed-loader.js

```diff
@@ -0,0 +1,91 @@
+var FEED_URL = 'https://blog.webmaker.org/tag/teachtheweb/feed';
+var DEFAULT_CONTENT_SNIPPET_WORD_COUNT = 70;
+var SCRIPT_SRC = 'https://www.google.com/jsapi';
+
+// http://stackoverflow.com/a/822464
+var HTML_TAG_RE = /<(?:.|\n)*?>/gm;
+
+var scriptLoading = false;
+var scriptLoadCallbacks = [];
```

Comment:
Hmm, I guess it's actually not *really* needed unless multiple requests to fetch the blog posts are issued almost simultaneously, which is never the case in our current site--except on the manual test page if you click the button really fast with a slow internet connection--so it's technically over-engineering. But it was easier to just make it possible rather than document the limitation, I suppose. :stuck_out_tongue: 

Also, it could be useful if we decide to use other Google JS API features in the future *other* than just the feed API, so I guess there's that. e.g., imagine if we had another component below the blog which used some other Google JS API feature--both components would request the JS API at the same time, but we'd want to ensure that only one `<script>` tag was added to the DOM, and we'd want the callbacks given to both components to be called once that script was loaded.